### PR TITLE
NIFI-4081 - Added raw message option in GrokReader

### PR DESCRIPTION
**Pull request by pvillard31**
Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [X] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [X] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [X] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [X] Is your initial contribution a single, squashed commit?

### For code changes:
- [X] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [X] Have you written or updated unit tests to verify your changes?
- [X] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/grok/GrokRecordReader.java [hunk 1]
```diff
@@ -89,6 +93,7 @@ public Record nextRecord() throws IOException, MalformedRecordException {
         // the next call to nextRecord()) or we will attach it to the previously read record.
         String stackTrace = null;
         final StringBuilder toAppend = new StringBuilder();
+        final StringBuilder raw = new StringBuilder(line);
```
Comment (bbende) [inline nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/grok/GrokRecordReader.java]: I think this needs to be updated slightly based on some changes in master, looks like the beginning of this method changed a little bit and the 'line' variable is no longer visible here

**Comment by pvillard31** [comment]
Thanks @bbende, I updated my PR with latest master code.

**Comment by trixpan** [comment]
@pvillard31 I accept it is a bit of bit picking but is there any chance we call it _raw? IIRC this is the name we used on ParseCEF. Naming conventions make user experience smoother 😀

**Comment by pvillard31** [comment]
Hey @trixpan, consistency is always a good thing and thanks for spotting that. I changed the name to ``_raw``. Let me know if something else is needed.

**Comment by markap14** [comment]
@pvillard31 looks like this PR has some conflicts. Do you mind rebasing?

**Comment by pvillard31** [comment]
@markap14 done and updated a bit to match the recent refactoring.

**Comment by markap14** [comment]
@pvillard31 all looks good from my end. +1 merged to master.

**Pull request closed without merging**